packages/cosmic-swingset/lib/ag-solo/vats/lib-board.js

```diff
@@ -16,7 +38,8 @@ function makeBoard(seed = 0) {
         // Retry until we have a unique id.
         let id;
         do {
-          id = sparseInts.next().value.toString();
+          const num = sparseInts.next().value;
+          id = `${num.toString()}${calcCrc(num)}`;
```

Comment:
Curious why `${num.toString()}` instead of `${num}`?

Reply:
Cargo culting. :smile:

It's fixed now.